apk-generator/v2/app/utils/notification.py

```diff
@@ -46,7 +81,24 @@ def send_mail_via_smtp_(config, payload):
         mailer.start()
         message = Message(author=payload['from'], to=payload['to'])
         message.subject = payload['subject']
-        message.plain = strip_tags(payload['html'])
-        message.rich = payload['html']
+        message.plain = strip_tags(payload['message'])
+        message.rich = payload['message']
+        message.attach(payload['attachment'], data=None, maintype=None, subtype=None, inline=False)
```

Comment:
@mahikaw see here https://github.com/marrow/mailer/blob/develop/marrow/mailer/message.py#L254 ... The other arguments are not needed. The defaults are `None` anyway. So remove them please. Once this is done, I think the PR can be merged. 

Good work :)